dist/circle-progress.js

```diff
@@ -157,6 +157,13 @@ License: MIT
             canvas.width = this.size;
             canvas.height = this.size;
             this.ctx = canvas.getContext('2d');
+
+            if (window.devicePixelRatio > 1) {
+                var scaleBy = window.devicePixelRatio;
+                canvas.style.width = canvas.style.height = [this.size, 'px'].join('');
```

Comment:
I've read that current JS implementations do `123 + 'px'` as fast as `[123, 'px'].join('')`. Let's keep the code simple ;)